ACE/ace/os_include/os_pthread.h

```diff
@@ -45,9 +45,13 @@
 
 
 #if !defined (ACE_LACKS_PTHREAD_H)
+# if !defined ACE_VXWORKS || ACE_VXWORKS < 0x700
```

Comment:
Maybe add a macro which is set in the os config file, hopefully with a next vxworks version we can then undefine that